ui/src/data-products/kilometer-lengths/entire-rail-network-km-lengths-listing.tsx

```diff
@@ -17,7 +18,9 @@ export const EntireRailNetworkKmLengthsListing = () => {
             </p>
             <PrivilegeRequired privilege={DOWNLOAD_GEOMETRY}>
                 <div className={styles['data-products__search']}>
-                    <a qa-id="km-lengths-csv-download" href={getEntireRailNetworkKmLengthsCsvUrl()}>
+                    <a
+                        qa-id="km-lengths-csv-download"
+                        href={getEntireRailNetworkKmLengthsCsvUrl(officialMainLayoutContext())}>
```

Comment:
Onkohan tietotuotteissa tarpeen ottaa branchi huomioon, vai riittäisiköhän aina vaan oletus että ne tehdään main-officialista 🤔

Reply:
Joo tuo on ihan hyvä kysymys. Varmaan ainakin isossa osassa niistä official riittänee, mutta en näkisi mahdottomana että niitä haluttaisi jossain kohtaa suunnitelmistakin jotta saadaan koosteita myös tulevasta tavarasta. Laitoin ne nyt kuitenkin tänne api:in tarjolle koska se oli aika lailla helppoa ja nuo ei taida tehdä paljon omia sql-hakujaan vaan menee niillä yleisillä (joille tuo design-puolen haku pitää toteuttaa joka tapauksessa).

Jos tuosta tulee hankaluuksia toteutuksen puolella niin ehdottomasti ihan validia pakittaa nämä ainakin v1:stä pois.